pages/register.js

```diff
@@ -0,0 +1,65 @@
+import { useRouter } from "next/router";
+import { useDispatch } from "react-redux";
+import { signIn } from "../actions";
+import CustomForm from "../components/CustomForm";
+import LeftSignUpCard from "../components/LeftSIgnUpCard";
+import SignUpStyle from "../components/styles/SignupStyle";
+
+export default function UserRegisterPage() {
+  const router = useRouter();
+  const dispatch = useDispatch();
+  const formArray = [
```

Comment:
move this to a constant file specific to register page.

Reply:
done